### DISCUSSION TITLE
airflow dags ui why not paging display ?

### DISCUSSION THREAD

**Discussion by chuxiangfeng** [apache/airflow#31995] [Q&A] [answered]
in dags ui list，no matter how many there are，it always display all without page，can any way to page display？

**Comment by chuxiangfeng** [comment]
can any one help?

**Reply by potiuk** [reply, marked as answer]
Maybe documentation can help? 

Have you seen and tried https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#page-size

Or maybe you have an old version that does not support it? Who knows - you have not included any screenshots or information about the version you use. It would have been immensely useful to save time of people who you ask for help in the future to provide useful information that will make them eagee to help you in their free time. 

What helps is to spend a bit more time on your side on asking your question in the way that will make it easy for somene trying to help to get the context you are in.
